### PR TITLE
Add Ant Design Icons

### PR DESCRIPTION
**Pull request by afc163**
- https://ant.design/components/icon/
- http://github.com/ant-design/ant-design/

anticon

### CODE REVIEW

**Comment by keremciu** [comment]
@afc163 amazing contribution man! nice to see this icon set here. I had a small request, if you can handle that I'd like to merge this one 🎉

**Comment by afc163** [comment]
Tell me about it. 😃

File: fonts.json [hunk 1]
```diff
@@ -15,6 +15,10 @@
     "simple-line-icons" : {
       "path" : "simple-line-icons.json",
       "container" : "<i class='icon-*****'></i>"
+    },
+    "anticon" : {
+      "path" : "anticon.json",
+      "container" : "<i class='anticon anticon-*****'></i>"
```
Comment (keremciu) [inline fonts.json]: @afc163

**Comment by keremciu** [comment]
I sent the review above

**Comment by keremciu** [comment]
merged 🎉 @afc163

**Comment by afc163** [comment]
🎉

**Pull request merged**